recipes/go-sops/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set import_path = "github.com/mozilla/sops" %}
+{% set version = "3.5.0" %}
+
+{% set name = import_path.split('/')[-1] %}
+{% set pkg_src = ('src/'+import_path).replace("/", os.sep) %}
+
+package:
+  name: go-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  - folder: {{ pkg_src }}
+    url: https://{{ import_path }}/archive/v{{version}}.tar.gz
+    sha256: a9c257dc5ddaab736dce08b8c5b1f00e6ca1e3171909b6d7385689044ebe759b
+
+build:
+  number: 0
+  script:
+    - pushd {{ pkg_src }}
+    - make install
+ 
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('go') }}
+    - make
```

It seems that broke windows:
```suggestion
    - make  # [unix]
```